Cargo.toml

```diff
@@ -40,6 +39,9 @@ encrypted_transfers = { version = "*", path = "./concordium-base/rust-src/encryp
 eddsa_ed25519 = { version = "*", path = "./concordium-base/rust-src/eddsa_ed25519/" }
 concordium-contracts-common = { version = "*", path = "./concordium-contracts-common/", features = ["derive-serde"]}
 random_oracle = { version = "*", path = "./concordium-base/rust-src/random_oracle/" }
+num-bigint = "0.4.3"
+num-traits = "0.2.15"
```

Comment:
It's probably better to not specify patch versions unless there is a specific reason to do it.